scripts/variantstore/docs/aou/vds/removing_samples.md

```diff
@@ -0,0 +1,152 @@
+# Removing samples from the AoU VDS
+
+The following example walks through the procedure we followed for removing AI/AN samples from a preliminary version of
+the Delta VDS in November 2022. Because we did not have to re-create the filter with these samples removed, we did not
```

Comment:
Did you do this already? I had thought the documentation was being written up so that the AI/AN samples can be withdrawn from the Delta VDS. In any case, maybe this should be written more generically - that is don't reference the specific example.

Reply:
Right, this is written in the past tense but it actually hasn't been done yet.

I'm wary of trying to write something generically as the cluster configuration parts of this definitely are specific to Delta. We did want to document Delta cluster configuration as that's going to be required for the AI/AN removal work, but Delta configuration might not be appropriate for Echo or beyond.

Given the unpredictably of our requirements changes I'd also be reluctant to guarantee the approach here would work for every future removal scenario. e.g. we did have an approach for removing samples that we used for Dragen equivalent removal, but that approach did not generalize to AI/AN removal due to the number of samples being removed.

Anyway, happy to discuss further.

Reply:
No, that makes sense. I guess I was mostly confused about the past tense. I understand about the cluster configuration.